### ISSUE TITLE
Feature Flag Part 2

### ISSUE THREAD

**Issue by afiune** [chef/chef-workstation#724]
### Description
Splitting https://github.com/chef/chef-workstation/issues/609 up into a new issue to
continue its work.

This new Issue includes:
* (vision-to-the-future) Through a UI coming from our Chef Workstation App: https://github.com/chef/chef-workstation-app/pull/125

**Comment by tyler-ball** [comment]
We're going to get the hover text and then this will be unblocked and ready to be finished

**Comment by tyler-ball** [comment]
We're splitting this. The UI in the app is going back into the parking lot, and the documentation about the toml file is moving forward.

The toml file is hard for users to find and interact with but we do not currently have the resources to complete the UX based work, so we're getting what we have merged.
